modules/fluctAnalyticsAdapter.js

```diff
@@ -12,26 +12,27 @@ import {
   deepClone,
 } from '../src/utils.js'
 import find from 'core-js-pure/features/array/find.js'
+import $$PREBID_GLOBAL$$ from '../src/prebid.js'
 
 const url = 'https://an.adingo.jp'
 
 /** @typedef {{ad: string, adId: string, adUnitCode: string, adUrl: string, adserverTargeting: any, auctionId: string, bidder: string, bidderCode: string, cpm: number, creativeId: string, currency: string, dealId: string, height: number, mediaType: string, netRevenue: boolean, originalCpm: number, originalCurrency: string, params: any, pbAg: string, pbCg: string, pbDg: string, pbHg: string, pbLg: string, pbMg: string, pbLg: string, requestId: string, requestTimestamp: number, responseTimestamp: number, size: string, source: string, status: string, statusMessage: string, timeToRespond: string, ttl: string, width: number, bidId?: string}} BidResponse */
 /** @typedef {{bidder: string, params: any}} Bid */
-/** @typedef {{code: string, _code?: string, analytics?: {bidder: string, dwid: string}[], bids?: Bid[], mediaTypes: {banner: {name: string, sizes: number[][]}}, sizes: number[][], transactionId: string}} AdUnit */
+/** @typedef {{code: string, _code?: string, path?: string, analytics?: {bidder: string, dwid: string}[], bids?: Bid[], mediaTypes: {banner: {name: string, sizes: number[][]}}, sizes: number[][], transactionId: string}} AdUnit */
 /** @typedef {{adUnitCodes: string[], adUnits: AdUnit[], auctionEnd: number, auctionId: string, auctionStatus: string, bidderRequests: any[], bidsReceived: BidResponse[], labels?: string, noBids: BidResponse[], timeout: number, timestamp: number, winningBids: BidResponse[], bids: {[key: string]: BidResponse}[]}} PbAuction */
 /** @typedef {{registered: boolean}} Gpt */
 /** @typedef {{[key: string]: string }} Slots */
 
 /** @type {{auctions: {[auctionId: string]: PbAuction}, adUnits: {[adUnitCode: string]: AdUnit}, gpt: Gpt, timeouts: {[auctionId: string]: number}}} */
 const cache = {
   auctions: {},
-  adUnits: {},
   gpt: {},
   timeouts: {},
-}
+};
+$$PREBID_GLOBAL$$.cache = cache;
```

Comment:
prebidのインスタンスにcacheがいないかが気になり

📝  いないので大丈夫
![スクリーンショット 2022-02-22 19 28 47](https://user-images.githubusercontent.com/32355076/155113775-a4889e85-3bb1-42a2-a46e-b5f4d2a92629.png)

Reply:
グローバルなオブジェクトにcacheを生やして嬉しい理由がよくわかっていない

Reply:
デバッグ用ですね。様々な理由でブラウザdebugger使うのが困難な場合があり。

Reply:
理解ですー

あんまりケースとしてはなさそうだけど、Prebid本家でcacheというプロパティが使われ始めたときに死ぬので、一段挟んでおいてもよさそうと思いますた。こんな感じで

```suggestion
$$PREBID_GLOBAL$$.fluct = {cache: cache};
```

Reply:
確かに！